### PR TITLE
Set up testing environment

### PR DESCRIPTION
**Pull request by raptorf1**
# [PT Story](https://www.pivotaltracker.com/story/show/165794204)

* Adds cucumber gem
* Installs cucumber with rails
* Adds database_cleaner gem
* Removes coffee-rails gem
* Adds disablers for rails generators

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -43,6 +41,8 @@ group :development, :test do
   gem 'pry-rails'
   gem 'rspec-rails'
   gem 'shoulda-matchers'
+  gem 'cucumber-rails', require: false
+  gem 'database_cleaner'
 end
```
Comment (faraznaeem) [inline Gemfile:26]: Please clean out the comments

File: Gemfile [hunk 2]
```diff
@@ -43,6 +41,8 @@ group :development, :test do
   gem 'pry-rails'
```
Comment (faraznaeem) [inline Gemfile:18]: do you need both byebug and pry-rails?

**Review by oliverochman — COMMENTED** [review]
I would suggest adding a javascript driver to the testing setup

**Review by faraznaeem — DISMISSED** [review]
I have only seen PR by @raptorf1 I would like to see PRs by other members as well. Now as you have a codebase up

**Review by oliverochman — COMMENTED** [review]
Next time I would like to see more commits

File: Gemfile (hunk 1 above)
Comment (raptorf1) [reply Gemfile:26]: done

File: Gemfile (hunk 2 above)
Comment (raptorf1) [reply Gemfile:18]: no. we had that discussion already :P

**Comment by raptorf1** [comment]
> I would suggest adding a javascript driver to the testing setup

added

**Review by oliverochman — APPROVED** [review]
Good job guys!

**Pull request merged**